packages/repository/src/type-resolver.ts

```diff
@@ -14,10 +14,10 @@ import {Class} from './common-types';
  * the actual reference to the class itself until later, when both sides
  * of the relation are created as JavaScript classes.
  *
- * @template Type The type we are resolving, for example `Entity` or `Product`.
+ * @typeparam Type The type we are resolving, for example `Entity` or `Product`.
```

Comment:
Does `strong-docs` recognize `@typeparam` annotation? If not, then I think this change is going to break API docs that we are generating right now :(

Reply:
TypeDoc already uses `@typeparam` - https://github.com/TypeStrong/typedoc/blob/0014556908f807486d2c1d17812ed472113fe522/examples/basic/src/generics.ts#L4